docker/openemr/flex-edge/openemr.sh

```diff
@@ -340,6 +328,7 @@ if [ "$CONFIG" == "1" ] &&
         rm -f sql_patch.php
         rm -f sql_upgrade.php
         rm -f ippf_upgrade.php
+        rm -f auto_configure.php
```

Comment:
Done.

Reply:
There's something odd in that stanza. It runs if `/var/www/localhost/htdocs/auto_configure.php` exists (note path) but it deletes the file found in `/var/www/localhost/htdocs/openemr/` (note path again) before deleting the file `/var/www/localhost/htdocs/auto_configure.php`. This isn't /duplicative/, but is one or the other of these just flat wrong?

Reply:
Looks like the production line dockers use /var/www/localhost/htdocs/openemr/auto_configure.php while the development line dockers use /var/www/localhost/htdocs/auto_configure.php which I am pretty sure was done since there is no openemr directory yet during docker building in flex series.
So everything in the script should be using /var/www/localhost/htdocs/auto_configure.php

Reply:
Done.